packages/react-storage/src/components/StorageBrowser/views/LocationActionView/CopyFilesControls.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+
+import { ButtonElement, ViewElement } from '../../context/elements';
+
+import { Controls } from '../Controls';
+
+import { Title } from './Controls/Title';
+import { displayText } from '../../displayText/en';
+import { CLASS_BASE } from '../constants';
+import { DestinationPicker } from './DestinationPicker';
+
+import { useCopyView } from './CopyView/useCopyView';
+import { DataTableControl } from '../../controls/DataTableControl';
+import { ControlsContextProvider } from '../../controls/context';
+import { getActionViewTableData } from './utils';
+import { useStore } from '../../providers/store';
+import { ControlsContext } from '../../controls/types';
+import { ActionStartControl } from '../../controls/ActionStartControl';
+import { getTasksHaveStarted } from './utils';
+import { DescriptionList } from '../../components/DescriptionList';
+import { StatusDisplayControl } from '../../controls/StatusDisplayControl';
+import { getDestinationListFullPrefix } from './utils/getDestinationPickerDataTable';
+
+const { Exit } = Controls;
+const { actionSetDestination } = displayText;
+
+export const CopyFilesControls = ({
```

Comment:
Can you please just add this to the `CopyView` scaffold component and use the props that are declared for it? We don't need these "Controls" files

Reply:
🍕

Reply:
⭐ suck